src/uct/tcp/tcp_ep.c

```diff
@@ -2067,17 +2073,28 @@ ucs_status_t uct_tcp_ep_flush(uct_ep_h tl_ep, unsigned flags,
     }
 
     status = uct_tcp_ep_check_tx_res(ep);
-    if (status == UCS_ERR_NO_RESOURCE) {
-        UCT_TL_EP_STAT_FLUSH_WAIT(&ep->super);
```

Comment:
why removed?

Reply:
this was a bug. we count `FLUSH_WAIT` if we return `UCS_INPROGRESS`
now it is the same as we do in IB transports